src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -1299,6 +1301,8 @@ namespace ReportSizingManager {
 						} else if ( OASysEqSizing( CurOASysNum ).HeatingAirFlow ) {
 							// Parent object sets heating flow rate
 							AutosizeDes = OASysEqSizing ( CurOASysNum ).HeatingAirVolFlow;
+						} else if ( DataDesicRegCoil ) {
+							AutosizeDes = FinalSysSizing( CurSysNum ).DesMainVolFlow;
```

Comment:
Placing this else if after others raises the issue of priority. If DataDesicRegCoil is required to override all other flags, it should be first in the list of tests.

Why is an OA system component using DesMainVolFlow ? Keep in mind that the others are set by a parent, you don't know what the OASysEqSizing().HeatingAirFlow represents here, only that the parent told you to use it.